config/cobbler/distro_signatures.json

```diff
@@ -445,6 +445,22 @@
     "kernel_options_post":"",
     "boot_files":[],
     "boot_loaders":{"ppc64el":["grub2"]}
+   },
+   "xenial": {
+    "signatures":["dists", ".disk"],
+    "version_file":"Release|mini-info",
+    "version_file_regex":"Archive: xenial",

```

Comment:
can't find "Archive: xenial" in `Release`, or `mini-info`.
should be: `"version_file_regex":"Codename: xenial|Ubuntu 16.04",`